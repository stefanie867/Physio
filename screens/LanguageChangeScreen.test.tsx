import React from 'react';
import renderer from 'react-test-renderer';

import LanguageChangeScreen from './LanguageChangeScreen';

describe('LanguageChangeScreen tests', () => {
  it('Snapshot test', () => {
    const tree = renderer.create(<LanguageChangeScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});