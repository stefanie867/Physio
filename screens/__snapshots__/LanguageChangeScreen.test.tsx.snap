// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LanguageChangeScreen tests Snapshot test 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
      },
      Object {
        "flex": 1,
      },
    ]
  }
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "borderBottomWidth": 0.5,
          "borderColor": "#bcbbc1",
          "flexDirection": "row",
          "padding": 14,
        }
      }
      testID="padView"
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "borderRadius": 17,
            "height": 34,
            "width": 34,
          }
        }
        theme={
          Object {
            "colors": Object {
              "black": "#242424",
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "grey": "rgba(0, 0, 0, 0.54)",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "grey": "#7d7d7d",
                  "primary": "#007aff",
                  "searchBg": "#dcdce1",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#ca71eb",
              "success": "#52c41a",
              "warning": "#faad14",
              "white": "#ffffff",
            },
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "transparent",
              "borderRadius": 17,
              "flex": 1,
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <Image
            borderRadius={17}
            onLoad={[Function]}
            source={1}
            style={
              Object {
                "bottom": 0,
                "flex": 1,
                "height": null,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": null,
              }
            }
            testID="RNE__Image"
            theme={
              Object {
                "colors": Object {
                  "black": "#242424",
                  "disabled": "hsl(208, 8%, 90%)",
                  "divider": "#bcbbc1",
                  "error": "#ff190c",
                  "grey0": "#393e42",
                  "grey1": "#43484d",
                  "grey2": "#5e6977",
                  "grey3": "#86939e",
                  "grey4": "#bdc6cf",
                  "grey5": "#e1e8ee",
                  "greyOutline": "#bbb",
                  "platform": Object {
                    "android": Object {
                      "error": "#f44336",
                      "grey": "rgba(0, 0, 0, 0.54)",
                      "primary": "#2196f3",
                      "secondary": "#9C27B0",
                      "success": "#4caf50",
                      "warning": "#ffeb3b",
                    },
                    "ios": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                  },
                  "primary": "#2089dc",
                  "searchBg": "#303337",
                  "secondary": "#ca71eb",
                  "success": "#52c41a",
                  "warning": "#faad14",
                  "white": "#ffffff",
                },
              }
            }
            transition={true}
          />
          <View
            accessibilityElementsHidden={true}
            importantForAccessibility="no-hide-descendants"
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#bdbdbd",
                  "flex": 1,
                  "height": null,
                  "justifyContent": "center",
                  "width": null,
                }
              }
              testID="RNE__Image__placeholder"
            />
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "height": null,
                "width": null,
              }
            }
          />
        </View>
      </View>
      <View
        style={
          Object {
            "paddingLeft": 16,
          }
        }
      />
      <Text
        onPress={[Function]}
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#242424",
            "fontSize": 16,
            "width": "100%",
          }
        }
        testID="listItemTitle"
      >
        English
      </Text>
    </View>
  </View>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "borderBottomWidth": 0.5,
          "borderColor": "#bcbbc1",
          "flexDirection": "row",
          "padding": 14,
        }
      }
      testID="padView"
    >
      <View
        style={
          Object {
            "backgroundColor": "transparent",
            "borderRadius": 17,
            "height": 34,
            "width": 34,
          }
        }
        theme={
          Object {
            "colors": Object {
              "black": "#242424",
              "disabled": "hsl(208, 8%, 90%)",
              "divider": "#bcbbc1",
              "error": "#ff190c",
              "grey0": "#393e42",
              "grey1": "#43484d",
              "grey2": "#5e6977",
              "grey3": "#86939e",
              "grey4": "#bdc6cf",
              "grey5": "#e1e8ee",
              "greyOutline": "#bbb",
              "platform": Object {
                "android": Object {
                  "error": "#f44336",
                  "grey": "rgba(0, 0, 0, 0.54)",
                  "primary": "#2196f3",
                  "secondary": "#9C27B0",
                  "success": "#4caf50",
                  "warning": "#ffeb3b",
                },
                "ios": Object {
                  "error": "#ff3b30",
                  "grey": "#7d7d7d",
                  "primary": "#007aff",
                  "searchBg": "#dcdce1",
                  "secondary": "#5856d6",
                  "success": "#4cd964",
                  "warning": "#ffcc00",
                },
              },
              "primary": "#2089dc",
              "searchBg": "#303337",
              "secondary": "#ca71eb",
              "success": "#52c41a",
              "warning": "#faad14",
              "white": "#ffffff",
            },
          }
        }
      >
        <View
          accessibilityIgnoresInvertColors={true}
          style={
            Object {
              "backgroundColor": "transparent",
              "borderRadius": 17,
              "flex": 1,
              "overflow": "hidden",
              "position": "relative",
            }
          }
        >
          <Image
            borderRadius={17}
            onLoad={[Function]}
            source={1}
            style={
              Object {
                "bottom": 0,
                "flex": 1,
                "height": null,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
                "width": null,
              }
            }
            testID="RNE__Image"
            theme={
              Object {
                "colors": Object {
                  "black": "#242424",
                  "disabled": "hsl(208, 8%, 90%)",
                  "divider": "#bcbbc1",
                  "error": "#ff190c",
                  "grey0": "#393e42",
                  "grey1": "#43484d",
                  "grey2": "#5e6977",
                  "grey3": "#86939e",
                  "grey4": "#bdc6cf",
                  "grey5": "#e1e8ee",
                  "greyOutline": "#bbb",
                  "platform": Object {
                    "android": Object {
                      "error": "#f44336",
                      "grey": "rgba(0, 0, 0, 0.54)",
                      "primary": "#2196f3",
                      "secondary": "#9C27B0",
                      "success": "#4caf50",
                      "warning": "#ffeb3b",
                    },
                    "ios": Object {
                      "error": "#ff3b30",
                      "grey": "#7d7d7d",
                      "primary": "#007aff",
                      "searchBg": "#dcdce1",
                      "secondary": "#5856d6",
                      "success": "#4cd964",
                      "warning": "#ffcc00",
                    },
                  },
                  "primary": "#2089dc",
                  "searchBg": "#303337",
                  "secondary": "#ca71eb",
                  "success": "#52c41a",
                  "warning": "#faad14",
                  "white": "#ffffff",
                },
              }
            }
            transition={true}
          />
          <View
            accessibilityElementsHidden={true}
            importantForAccessibility="no-hide-descendants"
            pointerEvents="none"
            style={
              Object {
                "bottom": 0,
                "left": 0,
                "opacity": 1,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#bdbdbd",
                  "flex": 1,
                  "height": null,
                  "justifyContent": "center",
                  "width": null,
                }
              }
              testID="RNE__Image__placeholder"
            />
          </View>
          <View
            style={
              Object {
                "flex": 1,
                "height": null,
                "width": null,
              }
            }
          />
        </View>
      </View>
      <View
        style={
          Object {
            "paddingLeft": 16,
          }
        }
      />
      <Text
        onPress={[Function]}
        style={
          Object {
            "backgroundColor": "transparent",
            "color": "#242424",
            "fontSize": 16,
            "width": "100%",
          }
        }
        testID="listItemTitle"
      >
        German
      </Text>
    </View>
  </View>
</View>
`;
